frontend/src/components/@shared/Calendar/SelectedDate/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+import { ScheduleType } from 'types/data';
+import { parsedTime } from 'utils/date';
+
+import * as S from './index.styled';
+
+interface DateProps {
+  date: number;
+  schedule: ScheduleType;
+}
+
+function SelectedDate({ date, schedule }: DateProps) {
+  const [isHovering, setIsHovering] = useState(false);
+
+  const toggleHovering = (isHovering: boolean) => () => {
+    setIsHovering(isHovering);
```

Comment:
`isHovering` 도 알아볼 수 있는 네이밍이지만, `isHover`만 해도 깔끔할 것 같습니다!

```suggestion
  const [isHover, setIsHover] = useState(false);

  const toggleHover = (isHover: boolean) => () => {
    setIsHover(isHover);
```